src/context-items/custom-validation.ts

```diff
@@ -17,14 +17,19 @@ export class CustomValidation implements ContextItem {
       // A promise that was resolved only adds an error if negated.
       // Otherwise it always suceeds
       if ((!isPromise && failed) || (isPromise && this.negated)) {
-        context.addError(this.message, value, meta);
+        context.addError(this.message, value, meta, 'customValidator');
```

Comment:
How satisfied are you with `customValidator` here? 🤔

Reply:
No idea of how to improve🤔, maybe `.name` fallbacking to `'customValidator'`?